### ISSUE TITLE
Word "builds" is mention in Editors toolbar guide.

### ISSUE THREAD

**Issue by pszczesniak** [ckeditor/ckeditor5#16496]
## 📝 Provide a description of requested docs changes

[https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/getting-started/setup/toolbar.html#basic-toolbar-configuration](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/getting-started/setup/toolbar.html#basic-toolbar-configuration) below the infobox:

```
In the builds that contain toolbars, an optimal default toolbar set is defined for it.
```

Maybe some other word would suit better? :thinking:

**Comment by godai78** [comment]
That's one for a removal at all.

**Comment by godai78** [comment]
Fixed in 6dc93a5
